numpy/core/_internal.py

```diff
@@ -194,7 +197,17 @@ def _commastring(astr):
 
     return result
 
+class dummy_ctype(object):
+    def __init__(self, cls):
+        self._cls = cls
+    def __mul__(self, other):
+        return self
+    def __call__(self, other):
```

Comment:
Hmm, this might actually need to be `*other`, since we call it with `*` further down. Is there any easy way to test a lack of ctypes?